### ISSUE TITLE
The color contrast for the remix, view, love, and favorites icons in the projects page is not in line with WCAG standards for color contrast.

### ISSUE THREAD

**Issue by Paul-Clue** [LLK/scratch-www#7255]
### Where To Find Bug
`/projects`

### Expected Behaviour
People with low vision should be able to see the icons in grayscale properly.

### Actual Behaviour
The icons in grayscale are not WCAG compliant and, thus, are difficult for people with low vision to see.

### Steps To Reproduce

- Open https://scratch.mit.edu/ in any browser on your computer
- Click on any project
- Notice the show, remix, love, and favorite icons under the project picture.

### System Details
macOs Ventura, Chrome Browser, VoiceOver (AT)

**Comment by benjiwheeler** [comment]
On second thought, this one is a bit wider in scope than #7137 because of its including the remix and view icons.
